#Array of SlotProperties data structure
SlotProperties[] array